src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -230,7 +230,7 @@ public function addXmlContent($content, $charset = 'UTF-8')
         $dom->validateOnParse = true;
 
         if ('' !== trim($content)) {
-            @$dom->loadXML($content, LIBXML_NONET);
+            @$dom->loadXML($content, LIBXML_NONET | LIBXML_PARSEHUGE);
```

Comment:
You are right in php 5.3.9
ext\libxml\libxml.c
```c
#if LIBXML_VERSION >= 20703
	REGISTER_LONG_CONSTANT("LIBXML_PARSEHUGE",	XML_PARSE_HUGE,			CONST_CS | CONST_PERSISTENT);
#endif 
```
So minimum libxml version actualy 2.7.3 not 2.7.0